manifests/hivemq-operator/manifests.sh

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env bash
+set -e
+
+IS_HELM_INSTALLED=$(which helm >/dev/null 2>&1 || echo "Helm is not installed")
```

Comment:
Nice one!

Reply:
Just tested this locally with `helm2` to provoke the error. Should work fine on Linux as well!